pkg/nvml/lib.go

```diff
@@ -149,9 +149,9 @@ var nvmlDeviceGetGridLicensableFeatures = nvmlDeviceGetGridLicensableFeatures_v1
 var nvmlEventSetWait = nvmlEventSetWait_v1
 var nvmlDeviceGetAttributes = nvmlDeviceGetAttributes_v1
 var nvmlComputeInstanceGetInfo = nvmlComputeInstanceGetInfo_v1
-var DeviceGetComputeRunningProcesses = deviceGetComputeRunningProcesses_v1
-var DeviceGetGraphicsRunningProcesses = deviceGetGraphicsRunningProcesses_v1
-var DeviceGetMPSComputeRunningProcesses = deviceGetMPSComputeRunningProcesses_v1
+var deviceGetComputeRunningProcesses = deviceGetComputeRunningProcesses_v1
```

Comment:
Maybe a silly question: Can you remind me why we don't have an `nvml` prefix here?

Reply:
The nvml "functions" in this block come from #defines in nvml.h.  The underlying versioned functions that get assigned to these variables are generated as part of the bindings with the `v1`, `v2`, `v3`, etc. prefixes and then assigned in `updateVersionedSymbols` below.

The difference with these functions is that we need to wrap the underlying `nvml*_v1`, `nvml*_v2`, etc. functions with some extra, version specific logic. Because of this, we create a level of indirection before assinging them back out to the "unversioned" variable.